src/components/SearchModal/CurrencyList/index.tsx

```diff
@@ -232,33 +240,35 @@ export default function CurrencyList({
   isAddressSearch,
 }: {
   height: number
-  currencies: Currency[]
+  currencies: CurrencyWithRestriction[]
   otherListTokens?: WrappedTokenInfo[]
   selectedCurrency?: Currency | null
-  onCurrencySelect: (currency: Currency, hasWarning?: boolean) => void
+  onCurrencySelect: (currency: CurrencyWithRestriction, hasWarning?: boolean) => void
   otherCurrency?: Currency | null
   fixedListRef?: MutableRefObject<FixedSizeList | undefined>
   showCurrencyAmount?: boolean
   isLoading: boolean
   searchQuery: string
   isAddressSearch: string | false
 }) {
-  const itemData: Currency[] = useMemo(() => {
+  const itemData: CurrencyWithRestriction[] = useMemo(() => {
     if (otherListTokens && otherListTokens?.length > 0) {
-      return [...currencies, ...otherListTokens]
+      return [...currencies /* , ...otherListTokens*/]
```

Comment:
There are like secondary lists that aren't immediately prioritized that it may show if there are no tokens in the primary lists available. Since we don't use other lists other than our own Mauve one, we don't want to use other lists.